java101/src/com/java101/HeroService.java

```diff
@@ -0,0 +1,52 @@
+package com.java101;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class HeroService {
+
+	public static void main(String[] params) {
+		List<Hero> listaBohaterow = new ArrayList<>();
```

Comment:
Miałem na myśli to, żeby w klasie Application zostawić start aplikacji, a do HeroService przenieść kod tworzenia bohatera ;d